schema.json

```
@@ -27,19 +43,19 @@
             "properties": {
                 "originURL": {
                     "type": "string",
-                    "description": "Blah"
+                    "description": "the Storage URL for the site"
```

Nit: Suggest we capitalize the first letters of all of these, like so:

```suggestion
                    "description": "The Storage URL for the site"
```